presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveQueries.java

```diff
@@ -571,6 +571,9 @@ public void testCreateTableAsSelect()
                 .setCatalogSessionProperty("hive", "collect_column_statistics_on_write", "false")
                 .build();
 
+        // Clean up if table tmp already exists.
+        dropTable("tmp");
```

Comment:
nit: It might be cleaner to wrap CTAS in a try-finally and drop table in the finally

```
try {
   getQueryRunner().execute(session, "CREATE TABLE tmp AS SELECT * FROM nation");
} finally {
   dropTable("tmp");
}
```